.circleci/config.yml

```diff
@@ -47,13 +47,28 @@ jobs:
       - run:
           name: Verify project configurations
           command: npm run verify-config
-      - run:
-          name: Build
-          command: npm run build
       - run:
           name: Test
           command: npm run test
 
+  build:
+    docker:
+      - image: cimg/base:stable
+    resource_class: medium+
+    steps:
+      - checkout
+      - node/install
+      - vault/get-secrets:
```

Comment:
Is getting secrets necessary here? I have limited context, so am just curious

Reply:
Not necessary for this particular step.